com.bambulab.BambuStudio.yml

```diff
@@ -228,6 +219,16 @@ modules:
         dest: external-packages/Blosc
         sha256: dcb48bf43a672fa3de6a4b1de2c4c238709dad5893d1e097b8374ad84b1fc3b3
 
+      - type: file
```

Comment:
Building more stuff in the BambuStudio module makes the whole build take longer if we need to rebuild, because all the deps will be rebuilt as well.

As it is, if the boost or any dependencies before that aren't changed, we'll be rebuilding BambuStudio and all of its internal deps. Building more stuff inside the BambuStudio module as part of its internal deps is not desirable.

Patches are also more difficult to apply to the BambuStudio internal deps than if we were building them separately.